Tribler/Test/Community/channel/test_channel_base.py

```diff
@@ -0,0 +1,20 @@
+from Tribler.Test.test_as_server import AbstractServer
+from Tribler.community.channel.community import ChannelCommunity
+from Tribler.dispersy.dispersy import Dispersy
+from Tribler.dispersy.endpoint import ManualEnpoint
+from Tribler.dispersy.member import DummyMember
+from Tribler.dispersy.util import blocking_call_on_reactor_thread
+
+
+class AbstractChannelCommunity(AbstractServer):
+
+    # We have to initialize Dispersy and the tunnel community on the reactor thread
+    @blocking_call_on_reactor_thread
+    def setUp(self):
```

Comment:
iirc, the `annotate` argument should be present here (take a look at the `setUp` method of the superclass)/